### PR TITLE
docs: changes merender to me-*render*

### PR DESCRIPTION
**Pull request by r17x**
Based on our discussion about "merender" and "me-render"

### CODE REVIEW

File: src/content/learn/describing-the-ui.md [hunk 1]
```diff
@@ -314,7 +314,7 @@ Baca **[Mengoper Props ke sebuah Komponen](/learn/passing-props-to-a-component)*
 
 </LearnMore>
 
-## Merender Secara Kondisional {/*conditional-rendering*/}
+## Me-render Secara Kondisional {/*conditional-rendering*/}
```
Comment (zainfathoni) [inline src/content/learn/describing-the-ui.md:317]:
```suggestion
## Me-*render* Secara Kondisional {/*conditional-rendering*/}
```

Shall we make this *italic* as well?

File: src/content/learn/describing-the-ui.md [hunk 2]
```diff
@@ -358,11 +358,11 @@ export default function PackingList() {
 
 <LearnMore path="/learn/conditional-rendering">
 
-Baca **[Merender Secara Kondisional](/learn/conditional-rendering)** untuk mempelajari cara merender konten secara kondisional.
+Baca **[Me-render Secara Kondisional](/learn/conditional-rendering)** untuk mempelajari cara me-*render* konten secara kondisional.
 
 </LearnMore>
 
-## Merender List {/*rendering-lists*/}
+## Me-render List {/*rendering-lists*/}
```
Comment (zainfathoni) [inline src/content/learn/describing-the-ui.md:365]:
```suggestion
## Me-*render* List {/*rendering-lists*/}
```

Shall we make this *italic* as well?

File: src/content/learn/describing-the-ui.md [hunk 3]
```diff
@@ -358,11 +358,11 @@ export default function PackingList() {
 
 <LearnMore path="/learn/conditional-rendering">
 
-Baca **[Merender Secara Kondisional](/learn/conditional-rendering)** untuk mempelajari cara merender konten secara kondisional.
+Baca **[Me-render Secara Kondisional](/learn/conditional-rendering)** untuk mempelajari cara me-*render* konten secara kondisional.
```
Comment (zainfathoni) [inline src/content/learn/describing-the-ui.md:361]:
```suggestion
Baca **[Me-*render* Secara Kondisional](/learn/conditional-rendering)** untuk mempelajari cara me-*render* konten secara kondisional.
```

Shall we make this *italic* as well?

**Review by zainfathoni — CHANGES_REQUESTED** [review]

File: src/content/learn/describing-the-ui.md (hunk 1 above)
Comment (r17x) [reply src/content/learn/describing-the-ui.md:317]: Before, we have been discuss to NOT to make italic for **Heading** cc @resir014

File: src/content/learn/describing-the-ui.md (hunk 2 above)
Comment (r17x) [reply src/content/learn/describing-the-ui.md:365]: same reason at https://github.com/reactjs/id.react.dev/pull/503#discussion_r1181080945

File: src/content/learn/describing-the-ui.md (hunk 1 above)
Comment (mhaidarhanif) [reply src/content/learn/describing-the-ui.md:317]: Yes only in heading

**Review by mhaidarhanif — APPROVED** [review]
Another easy review!

File: src/content/learn/describing-the-ui.md (hunk 1 above)
Comment (r17x) [reply src/content/learn/describing-the-ui.md:317]: Waiting for @zainfathoni  confirm

**Review by mazipan — APPROVED** [review]

File: src/content/learn/describing-the-ui.md (hunk 1 above)
Comment (zainfathoni) [reply src/content/learn/describing-the-ui.md:317]: I see. I missed that guideline. If it has been decided, that's okay, then.
Though I'm wondering, what was the basis of that decision? Is it mentioned in PUEBI or anywhere else?

**Review by zainfathoni — APPROVED** [review]
LGTM 💯

**Pull request merged**